src/main/java/api/domain/pets/PetStatus.java

```diff
@@ -0,0 +1,5 @@
+package api.domain.pets;
+
+public enum PetStatus {
+    available, pending, sold;
```

Comment:
Instances of an enum are constants and should follow the conventions for constants.

In this case or continue to use enum but with uppercases: AVAILABLE, PENDING, SOLD or declare them as constants:

public static final String AVAILABLE = "available";
public static final String PENDING = "pending";
public static final String SOLD = "sold";

Reply:
Since `Status` is `ENUM` type (in Swagger Petstore), I've added the uppercase for the values.